packages/app-dynamic-pages/src/utils/getChildrenPaths.ts

```diff
@@ -0,0 +1,18 @@
+import { PbElement } from "~/types";
+
+export const getChildrenPaths = (elements: PbElement[] = []) => {
+    const paths: string[] = [];
+
+    const findDeep = (elements: PbElement[]) => {
```

Comment:
Can we have this defined outside of the function? so the function is not recreated every time?